src/editor/keybindings-definitions.ts

```diff
@@ -69,8 +69,8 @@ export const DEFAULT_KEYBINDINGS: Keybinding[] = [
         command: 'moveToPreviousPlaceholder',
     },
 
-    { key: '[Escape]', ifMode: 'math', command: ['switch-mode', 'command'] },
-    { key: '\\', ifMode: 'math', command: ['switch-mode', 'command'] },
+    { key: '[Escape]', ifMode: 'math', command: ['switchMode', 'command'] },
```

Comment:
Is there a reason for this change? Note that both "switchMode" and "switch-mode" are valid perform selectors. Or is this because of the stricter Typescript type checking for selectors?

Reply:
It is because of the stricter Typescript type checking. Typescript sadly can't do fancy string transforms.